WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -35,7 +35,7 @@ <h2>Help Pages</h2>
                 </div>
             </div>
         </div>
-    </div>
+    </div>-->
```

Comment:
Are we removing the filter buttons Ricky just put in too?  
They help filter sections by category quite effectively.

Reply:
I don't think the filter buttons work without search bar due to what the `JS` is running on. I can double check this before merging